fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/TombstoneImpl.java

```diff
@@ -58,4 +60,19 @@ public void delete() {
             throw new RepositoryRuntimeException(e);
         }
     }
+
+    @Override
+    public String toString() {
```

Comment:
`String` `+` has used a `StringBuilder` [under the hood](https://stackoverflow.com/questions/47605/string-concatenation-concat-vs-operator#47628) for a long time now. I think this would read better using `+=`.